src/main/java/in/projecteka/consentmanager/link/link/Link.java

```diff
@@ -138,8 +138,7 @@
                                 .map(user -> PatientLinksResponse.builder().
                                         patient(PatientLinks.builder()
                                                 .id(patientLinks.getId())
-                                                .firstName(user.getFirstName())
-                                                .lastName(user.getLastName())
+                                                .name(user.getName())
```

Comment:
can you check with Anusree, and remove name remove links if possible? and update the contract also?